impl/client/src/main/javascript/web/util/formatting.js

```diff
@@ -20,6 +20,9 @@
  */
 define("common-ui/util/formatting", ['common-ui/util/timeutil', 'common-ui/util/TextFormatter'], function(ReportTimeUtil, TextFormatter) {
   return {
+
+    const : totalDayMinutes = 1440,
```

Comment:
Please create this as a var, just before the return statement.
The way this is, you're defining a property named "const" in the exported object and defining a global variable called `totalDayMinutes`.

```js
var totalDayMinutes = 1440;

return {
  // ...
};
```